homeassistant/components/elevenlabs/__init__.py

```diff
@@ -3,6 +3,7 @@
 from __future__ import annotations
 
 from dataclasses import dataclass
+import logging
```

Comment:
logger not needed in this file

Reply:
Removed and also removed _LOGGER that was there.